pkg/logs/input/docker/launcher.go

```diff
@@ -192,6 +212,52 @@ func (l *Launcher) overrideSource(container *Container, source *config.LogSource
 	return newSource
 }
 
+// getFileSource create a new file source with the image short name if the source is ContainerCollectAll
+func (l *Launcher) getFileSource(container *Container, source *config.LogSource) *config.LogSource {
+	containerID := container.service.Identifier
+
+	// Just in case
+	if source.Name == config.ContainerCollectAll && l.collectAllSource == nil {
+		l.collectAllSource = source
+	}
+
+	standardService := l.serviceNameFunc(container.container.Name, dockerutil.ContainerIDToTaggerEntityName(containerID))
+	shortName, err := container.getShortImageName()
+
+	if err != nil {
+		log.Warnf("Could not get short image name for container %v: %v", ShortContainerID(containerID), err)
+	}
+
+	// If collect all may be update info for consistency with docker socket tailing
+	// source.UpdateInfo(containerID, fmt.Sprintf("Container ID: %s, Image: %s, Created: %s", ShortContainerID(containerID), shortName, container.container.Created))
```

Comment:
Should this be uncommented?

Reply:
I need to work on harmonization when tailing from file vs. when tailing from the Docker socket. Especially given the fact that tailing from file is done by emitting a new source to reuse the existing file logic. And that new source is appearing by itself in the status page. I don't think I will touch much of the tailing from socket execution path, but I think I will reuse the parent source logic you introduce to link the source create for the file and use that link to build a consistent status.